### PR TITLE
[qt5-base] fix feature mysql

### PR DESCRIPTION
**Pull request by autoantwort**
Instead of #28407

### CODE REVIEW

File: ports/qt5-base/portfile.cmake [hunk 1]
```diff
@@ -236,26 +233,22 @@ if(VCPKG_TARGET_IS_WINDOWS)
     else()
         list(APPEND CORE_OPTIONS -opengl dynamic) # other possible option without moving angle dlls: "-opengl desktop". "-opengel es2" only works with commented patch
     endif()
+    set(ADDITIONAL_WINDOWS_LIBS "ws2_32.lib secur32.lib advapi32.lib shell32.lib crypt32.lib user32.lib gdi32.lib")
```
Comment (dg0yt) [inline ports/qt5-base/portfile.cmake:236]: Use the maintainer function for asking pkgconfig...

Comment (autoantwort) [reply ports/qt5-base/portfile.cmake:236]: Can you elaborate? I only have moved some existing code

Comment (Neumann-A) [reply ports/qt5-base/portfile.cmake:236]: https://github.com/microsoft/vcpkg/blob/13ba94bc2416514ef45ca28a4ed6de6e0dc0dffa/ports/qt5-base/portfile.cmake#L200

Comment (autoantwort) [reply ports/qt5-base/portfile.cmake:236]: But mysql does not installs pkgconfig files

Comment (autoantwort) [reply ports/qt5-base/portfile.cmake:236]: @Neumann-A @dg0yt so should I change something or not?

Comment (Neumann-A) [reply ports/qt5-base/portfile.cmake:236]: > so should I change something or not?

I didn't requested a change I just quoted the line for the pkg-config stuff ;) But I don't think a change is necessary.

Comment (dg0yt) [reply ports/qt5-base/portfile.cmake:236]: Just go ahead.

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x64-windows
- x64-windows-static

**Review by vicroms — DISMISSED** [review]
LGTM once CI is green

**Pull request merged**